### DISCUSSION TITLE
Sync to QRZ, LotW, HRD, etc. when doing contest

### DISCUSSION THREAD

**Discussion by int2001** [magicbug/Cloudlog#2271] [Polls]
Cloudlog actually uploads QSOs to other services, when registered and the cronjob is running.
Either in realtime or as cron is scheduled.

This also happens when working in contest-mode.
Couldn't find anything that this could be against the rules.

But i'm interested how you are doing it.

**Comment by magicbug** [comment]
If using cloudlog for contesting i'm generally dabbling, but I'm only auto uploading to LoTW / Clublog.

at the end of the contest send a Cabrillo file to the contest organisers.
